### PR TITLE
Add setup instructions and a seash intro

### PR DESCRIPTION
**Pull request by aaaaalbert**

### CODE REVIEW

**Comment by aaaaalbert** [comment]
Please *squash* before you merge. The commit messages aren't really relevant, besides the fact that two new files are created.

File: seash_intro.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Other useful commands include:
+
+* `upload FILE` copies the file from the laptop to the sandbox.
+* `start UPLOADEDFILE` executes a file that was previously uploaded.
+* `download FILE` copies the file from the sandbox to the laptop.
+* `reset` removes all files from the sandbox, and clears the log.
+
+#### For Example
+
+Let's try out the `test_sensors.r2py` script that tests all sensor calls
```
Comment (lukpueh) [inline seash_intro.md:72]: Maybe we can link to the file here?
https://raw.githubusercontent.com/aaaaalbert/sensor-doodles/master/test_sensors.r2py

Comment (aaaaalbert) [reply seash_intro.md:72]: I'll add the sensor-doodles to the demokit I think. No linking required.

**Comment by lukpueh** [comment]
kk

File: seash_intro.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+device, and set it the active target for the following control commands.
+```
+ !> loadkeys user
+ !> as user
+ user@ !> browse
+ ['192.168.99.10:1224']
+Added targets: %1(192.168.99.10:1224:v1)
+Added group 'browsegood' with 1 target
+user@ !> on browsegood 
+user@browsegood !>
```
Comment (lukpueh) [inline seash_intro.md:49]: What do you think about splitting this section in three:

1. Load cryptographic keys
2 (a). Browse for NATed devices
3 (b). Contact devices in you network directly.

Comment (lukpueh) [reply seash_intro.md:49]: Should I merge and then change?

Comment (aaaaalbert) [reply seash_intro.md:49]: Fair suggestion, although it complicates the instructions if you (as an unexperienced reader) have to demultiplex what to do under which condition, when you don't know enough to know which condition you're operating under.

(I'd postpone this.)

**Review by lukpueh — APPROVED** [review]

**Pull request merged**